Source/WebCore/WebCoreSwift/CryptoKitShim.swift

```diff
@@ -0,0 +1,94 @@
+/*
+* Copyright (C) 2024 Apple Inc. All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions
+* are met:
+* 1. Redistributions of source code must retain the above copyright
+*    notice, this list of conditions and the following disclaimer.
+* 2. Redistributions in binary form must reproduce the above copyright
+*    notice, this list of conditions and the following disclaimer in the
+*    documentation and/or other materials provided with the distribution.
+*
+* THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
+* AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+* THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+* PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+* BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+* CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+* SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+* INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+* CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+* ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+* THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import Foundation
+import CryptoKit
+
+@objc(WKAesGcmErrorCodes)
+public enum AESGCMErrorCodes: Int {
+    case Success = 0
+    case WrongTagSize = 1
+    case EncryptionFailed = 2
+    case EncryptionResultNil = 3
+    case InvalidArgument = 4
+    case TooBigArguments = 5
+}
+
+@objc(WKAesGcm)
+public final class AesGcm: NSObject {
+    
+    @objc public static func encrypt(
+        _ key: UnsafePointer<UInt8>,
+        _ keySize: UInt,
+        _ iv: UnsafePointer<UInt8>,
+        _ ivSize: UInt,
+        _ additionalData: UnsafePointer<UInt8>,
+        _ additionalDataSize: UInt,
+        _ plaintext: UnsafePointer<UInt8>,
+        _ plaintextSize: UInt,
+        _ ciphertext: UnsafeMutablePointer<UInt8>) -> Int
```

Comment:
I would prefer having argument labels here. As is, the generated header is weird (lots of empty parameter names like `+[WKAesGcm encrypt:x :y :z ...]`), and since the types are all so primitive, it seems useful to have the lablels.

Reply:
done.